Test.c

```diff
@@ -4,11 +4,151 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Test.c
-// Program on branch main
+// Program on branch A5
 // ------------------------------------------------------------------------------------------------
-
+#pragma warning(disable:4996)
+#include "Header.h"
 #include <stdio.h>
+#include <stdbool.h>
+#include <malloc.h>
+#include <stdlib.h>
+#include <limits.h>
+
+#define MAX_SIZE 20
+
+typedef struct {
+   int Array[MAX_SIZE];
+   int Length;
+}Test;
+
+/// <summary>Checks if the array is sorted in ascending order.</summary>
+bool IsSorted (int arr[], int length) {
+   for (int i = 1; i < length; i++) if (arr[i - 1] > arr[i]) return false;
+   return true;
+}
+
+void PrintArray (int arr[], int size, int expLength) {
+   for (int i = 0; i < size; i++) printf ("%5d", arr[i]);
+   for (int i = expLength - size; i > 0; i--) printf ("%5s", "");
+}
+
+void PrintLine () {
+   printf ("--------------------------------------------------------------------------");
+   printf ("------------------------------------------------------------------------\n");
+}
+
+void TestSortAndSearch () {

```

Comment:
Test functions should ideally test one thing at a time. This allows us to pinpoint exact problems when they fail.